### ISSUE TITLE
Team-Friends (Submission)

### ISSUE THREAD

**Issue by mzminhaz5683** [Mahedi-61/MIU_Fall_2019#13]
First Submission:
Score : 0.80440

5th Submission:
Score : 0.80510

9th Submission
Score : 0.80650

10th Submission
Score : 0.80840

12th Submission
Score : 0.81310

14th Submission:
Score : 0.83820

15th Submission:
Score : 0.86160

16th Submission
Score : 0.87390

21th Submission
Score : 0.88910 (time: 4/12/19 : 20:01:26 GTM+ 0600)

**Comment by mzminhaz5683** [comment]
![Screenshot from 2019-11-26 08-15-10](https://user-images.githubusercontent.com/52498840/69593915-ebd9ad80-1024-11ea-9abc-bbc43a5f6af4.png)

Sir... I have more better score than 40th position... 0.86130 is his and mine is 0.86160. Then why I am not in the position of rank-40.....

**Comment by Mahedi-61** [comment]
This competition is closed. That's why it will not update for any new submission. The leaderboard reflects the final positions when the competition was open.

Don’t go over the moon thinking your model performs better than these submissions. Remember they submitted these models 5 years ago and many techniques which you are learning weren’t known by them.

**Comment by mzminhaz5683** [comment]
yes sir... I was only worry about if I have any mistake on submission so that there is no rank. As like before in the previous competition we got rank though the competition was over... i don't think anything as it's not all of my code. Baseline is yours and i have only some tricks to make that better.
